cmd/gateway/s3/gateway-s3.go

```diff
@@ -77,37 +77,37 @@ ENVIRONMENT VARIABLES:
 
 EXAMPLES:
   1. Start minio gateway server for AWS S3 backend.
-     {{.Prompt}} {{.EnvVarSetCommand}}(.*){{.AssignmentOperator}}accesskey
-     {{.Prompt}} {{.EnvVarSetCommand}}(.*){{.AssignmentOperator}}secretkey
+     {{.Prompt}} {{.EnvVarSetCommand}}MINIO_ACCESS_KEY{{.AssignmentOperator}}accesskey
```

Comment:
`{{.Prompt}} {{.EnvVarSetCommand}}MINIO_ACCESS_KEY{{.AssignmentOperator}}` needs to be `{{.Prompt}} {{.EnvVarSetCommand}} MINIO_ACCESS_KEY{{.AssignmentOperator}}`

Reply:
```suggestion
     {{.Prompt}} {{.EnvVarSetCommand}} MINIO_ACCESS_KEY{{.AssignmentOperator}}accesskey
```